packages/@aws-cdk/aws-iam/test/role.test.ts

```diff
@@ -4,6 +4,36 @@ import { Duration, Stack, App, CfnResource, RemovalPolicy, Lazy, Stage, DefaultS
 import { Construct } from 'constructs';
 import { AnyPrincipal, ArnPrincipal, CompositePrincipal, FederatedPrincipal, ManagedPolicy, PolicyStatement, Role, ServicePrincipal, User, Policy, PolicyDocument, Effect } from '../lib';
 
+describe('isRole', () => {
+  test('true if given Role instance', () => {
+    // GIVEN
+    const app = new App();
+    const stack = new Stack(app, 'MyStack');
+    // WHEN
+    const pureRole = new Role(stack, 'Role', {
+      assumedBy: new ServicePrincipal('sns.amazonaws.com'),
+    });
+
+    // THEN
+    expect(Role.isRole(pureRole)).toBe(true);
+  });
+
+  test('false if given IRole instance', () => {
```

Comment:
```suggestion
  test('false if given imported role instance', () => {
```

Role and imported Role are both `IRole` 🙂

Reply:
It is also very nice suggestion!!

I've commited this too!!